package.json

```diff
@@ -3,8 +3,10 @@
   "private": true,
   "version": "0.0.1",
   "scripts": {
-    "get-secrets": "ssh nest.bocoup.com \"cat /mnt/secrets/{rds,skillsbot-slack,skillsbot-db-dev}\" > ./.env",
-    "db": "source .env && PGPASSWORD=$DB_PASSWORD psql -h $DB_HOST -p $DB_PORT -U $DB_USER $DB_NAME",
+    "get-secrets": "ssh nest.bocoup.com cat /mnt/secrets/{skillsbot/slack-dev,rds-bocoup-db/{connection,db/skillsbot-dev}} > .env",
+    "db": "export $(<.env) && psql $PGNAME",
+    "db:create": "export $(<.env) && createdb $PGNAME",
+    "db:drop": "export $(<.env) && dropdb -i $PGNAME",
```

Comment:
this scares me a bit but as long as we can't accidentally do this to the production database, awesome